server/src/js/parent-helper.js

```diff
@@ -17,10 +17,27 @@ window.onmessage = function(e) {
     if (loaded) {
       doResize();
     }
-  } else if (e.data.scrollX) {
-    let frameOffset = document.getElementById("frame").getBoundingClientRect().top;
-    let toolbarHeight = document.getElementById("toolbar").clientHeight;
-    window.scrollTo(0, frameOffset + e.data.scrollY - toolbarHeight);
+  } else if (e.data.clipWidth) {
+    let frameOffset = document.getElementById("frame").getBoundingClientRect().top + window.scrollY,
```

Comment:
I switched to taking into account window.scrollY instead of using scrollTo(0,0).